### ISSUE TITLE
Different errors types that appear on the explorer UI and how they're triggered 

### ISSUE THREAD

**Issue by bugsbunnies** [ripple/explorer#189]
Looking for a list of possible errors that can appear on the explorer's UI, what they look like (can screenshot) and how they're triggered. This is to have a better understanding of the different error states and so we can make them more consistent in the long run, be more informative, allow users to take the appropriate actions, generally improve them to be more user friendly. They can also be added to a growing design system at Ripple.

**Comment by ckniffen** [comment]
@bugsbunnies are you asking about how tec codes are shown in the UI?

**Comment by bugsbunnies** [comment]
@ckniffen Yeah exactly, a list of all types of errors, how they look and why they happen
